GitUI/Translation/English.xlf

```diff
@@ -9023,17 +9023,28 @@ branch names, tag names)</source>
   </file>
   <file datatype="plaintext" original="RepoObjectsTree" source-language="en">
     <body>
-      <trans-unit id="_searchTooltip.Text">
-        <source>Search</source>
+      <trans-unit id="_filterForSelected.Text">
+        <source>&amp;Filter for selected</source>
+        <target />
+      </trans-unit>
+      <trans-unit id="_filterForSelectedRefsMenuItem.Text">
+        <source>&amp;Filter for selected</source>
         <target />
       </trans-unit>
-      <trans-unit id="_showBranchOnly.Text">
-        <source>Filter the revision grid to show this branch only
-To show all branches, right click the revision grid, select 'view' and then the 'show all branches'</source>
+      <trans-unit id="_filterForSelectedRefsMenuItem.ToolTipText">
+        <source>Filter the revision grid to show selected (underlined) refs (branches and tags) only.
+Hold Ctrl while clicking to de/select multiple and include descendant tree nodes by additionally holding Alt.
+To reset the filter, right click the revision grid, select 'View' and then 'Show all branches'.</source>
```

Comment:
"Show all branches" is available via the menu
![image](https://user-images.githubusercontent.com/4403806/158060537-bd361195-5cec-4c81-aec2-251802285b83.png)
```suggestion
You can reset the filter via View > Show all branches menu.</source>
```